e2e/cypress/integration/enterprise/cloud/billing/company_information_free_trial_spec.js

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+import {getRandomLetter} from '../../../../utils/index';
+
+describe('System Console - Company Information section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        gotoCompanyInformationScreen();
+    });
+
+    beforeEach(() => {
+        // # Click on close button of subscribe window if exist
+        cy.get('body').then(($body) => {
+            if ($body.find('.cancel-button').length > 0) {
+                cy.get('.cancel-button').click();
+            }
+        });
+    });
+
+    it('MM-37051 - Save Info button should not be enabled if any one of the mandetory feild is filled invalid data', () => {
+        const companyName = getRandomLetter(30);
+
+        // # click on Add Company Information button
```

Comment:
nit: Please start a comment with capital letter for consistency.
```suggestion
        // # Click on Add Company Information button
```